providers/statsig/src/test/java/dev/openfeature/contrib/providers/statsig/StatsigProviderTest.java

```diff
@@ -278,19 +278,15 @@ void getBooleanEvaluationByProperties() {
             evaluationContext).getValue());
     }
 
-    @SneakyThrows
     @Test
     void shouldThrowIfNotInitialized() {
         StatsigProviderConfig statsigProviderConfig = StatsigProviderConfig.builder().sdkKey("test").build();
         StatsigProvider tempstatsigProvider = new StatsigProvider(statsigProviderConfig);
 
         assertThrows(ProviderNotReadyError.class, ()-> tempstatsigProvider.getBooleanEvaluation(
         "fail_not_initialized", false, new ImmutableContext()));
-
-        OpenFeatureAPI.getInstance().setProviderAndWait("tempstatsigProvider", tempstatsigProvider);
     }
 
-    @SneakyThrows
```

Comment:
I think this actually hides the main issue (though you're right, its not needed to do this here). 

The main issue is that options can't be null, and the statsig sdk enforces that.

I pushed a change to the config file which I think will fix the actual issue

Reply:
I think if we're not going to add a brand new test to cover this failure case though, you might want to revert your test changes (because we do indeed need to default these options or the same error is thrown in production use).

Reply:
Thanks, yes you are correct. I will revert the changes keeping configuration relates change